instrumenter.go

```diff
@@ -130,17 +130,22 @@ func (i *instrumenter) markConds(n ast.Node) bool {
 	case *ast.UnaryExpr:
 		if n.Op == token.NOT {
```

Comment:
I'd like to keep the current behavior of gobco, as it provides a detailed and fine-grained coverage report, even though its name would have to change to gocco then, to properly account for the _condition coverage_ instead of the _branch coverage_, as detailed in https://en.wikipedia.org/wiki/Code_coverage. So instead of replacing the current test suite, there should be an alternate test suite that demonstrates the difference between branch coverage and condition coverage, and the mode should be selectable by the user via a command line flag.

Reply:
@rillig 
> My initial suggestion was to just remove the UnaryExpr and BinaryExpr cases. Didn't that work well enough?

It worked.

**New additions as per your suggestion:**
- Added Flag to instrument branch coverage
- Created separate test suite for bco and existing moved to coco
- updated test.sh to cover folders `testdata/instrumenter/coco` and `testdata/instrumenter/bco`

**_So now we should add flag `-want-c1` when need branch coverage as output_**